modules/nf-core/sourmash/compare/main.nf

```diff
@@ -14,16 +14,17 @@ process SOURMASH_COMPARE {
     val save_csv
 
     output:
-    tuple val(meta), path("*comp.npy.labels.txt"), path("*comp.npy"), optional:true, emit: matrix
-    path "*comp.csv"       , optional:true, emit: csv
-    path "versions.yml"    , emit: versions
+    tuple val(meta), path("*comp.npy")           , optional:true, emit: matrix
+    tuple val(meta), path("*comp.npy.labels.txt"), optional:true, emit: labels
```

Comment:
need to confirm if this will break anything down-stream or it is ok to separate out the outputs this way